guides/source/active_record_migrations.md

```diff
@@ -1307,3 +1623,118 @@ special comment like this:
 ```
 
  [Engines]: engines.html
+
+## Miscellaneous
+
+### Using UUIDs instead of IDs for Primary Keys
+
+By default, Rails uses auto-incrementing integers (often referred to as "IDs")
```

Comment:
I haven't seen ID referred in this way either and couldn't find any examples online.
ID is pretty ambiguous and can be non auto-incrementing as well I think.
This is what Wikipedia uses for [Primary Key](https://en.wikipedia.org/wiki/Primary_key)
> Informally, a primary key is "which attributes identify a record," and in simple cases constitute a single attribute: a unique ID.

Maybe we can avoid that and use something like the following instead?
```suggestion
### Using UUIDs instead of IDs for Primary Keys

By default, Rails uses auto-incrementing integers IDs
```

Reply:
Thanks for confirming @p8 and thanks for bringing up this discussion @jgaskins! I'll update accordingly!